src/components/Form/SetAclPermissionForm.jsx

```diff
@@ -110,34 +117,55 @@ const SetAclPermissionForm = () => {
       statusType="Permission status"
       defaultMessage="To be set..."
     >
-      <form onSubmit={handleAclPermission} autoComplete="off">
-        <div style={formRowStyle}>
-          <label htmlFor="set-acl-to">Set permissions to username: </label>
-          <br />
+      <Box display="flex" justifyContent="center">
+        <form onSubmit={handleAclPermission} autoComplete="off">
+          <FormControl>
+            <Typography htmlFor="set-acl-to">Set permissions to username:</Typography>
+            <TextField
+              id="set-acl-to"
+              name="setAclTo"
+              {...username}
+              placeholder={selectedUser}
+              label="Search username"
+              required
+            />
+          </FormControl>
+          <InputLabel id="set-acl-doctype">
```

Comment:
Like what Scott mentioned, I believe should be an `htmlFor` similar to how `<label>` uses `htmlFor` and uses `id` for the input fields.